### DISCUSSION TITLE
function definition (signature) in documentation

### DISCUSSION THREAD

**Discussion by st-clair-clarke** [rayepps/radash#40] [Ideas]
Hi, Thanks for impressive library.

1. I am requesting that the function signatures be placed on the same page with the function examples. It would aid my understanding much better.
2. Could we have a CHANGE.MD and a Whats New page in the repository to say what the changes are for an updated version and what is new for new changes?

Cheers

**Comment by sodiray** [comment]
I'll be adding argument docs next week/end.

For the changelog, I'll be adding a changelog file to the documentation site. I'll also be adding the ability to navigate to a previous version of the library in the docs.

It's a lot of work, might take me a bit. Does that cover what you're looking for?

**Comment by st-clair-clarke** [comment]
It does! Great! And thanks.
